src/mame/drivers/candela.cpp

```diff
@@ -0,0 +1,798 @@
+// license:BSD-3-Clause
+// copyright-holders:Joakim Larsson Edstrom
+/*
+ *
+ * History of Candela Data AB
+ *---------------------------
+ * The Candela computer was designed to be the big breakthough and developed by Candela Data AB, "a Didact Company".
+ * The Candela system was based around a main unit that could run OS-9 or Flex and a terminal unit that had a 
+ * propietary software including CDBASIC. The Candela system lost the battle of the swedish schools to
+ * the Compis computer by TeleNova which was based on CP/M initially.  Later both lost to IBM PC as we know.
+ * Candela Data continued to sell their system to the swedish industry without major successes despite great
+ * innovation and spririt.
+ *
+ * Misc links about the boards supported by this driver.
+ *-----------------------------------------------------
+ * http://frakaday.blogspot.com/p/candela.html
+ * http://elektronikforumet.com/forum/viewtopic.php?f=11&t=51424
+ *
+ *  TODO:
+ *  Candela designs:   can09t, can09
+ * -------------------------------------
+ *  - Add PCB layouts   OK
+ *  - Dump ROM:s,       OK      OK
+ *  - Keyboard
+ *  - Display/CRT
+ *  - Clickable Artwork
+ *  - Sound
+ *  - Cassette i/f
+ *  - Expansion bus
+ *  - Expansion overlay
+ *  - Interrupts
+ *  - Serial            OK
+ ****************************************************************************/
+
+#include "emu.h"
+#include "cpu/m6809/m6809.h"
+#include "machine/6821pia.h"
+#include "machine/6840ptm.h"
+#include "machine/6850acia.h"
+#include "machine/clock.h"
+#include "machine/ram.h"
+#include "machine/wd_fdc.h"
+#include "video/mc6845.h"
+
+// Features
+#include "imagedev/cassette.h"
+#include "bus/rs232/rs232.h"
+#include "screen.h"
+
+//**************************************************************************
+//  MACROS / CONSTANTS
+//**************************************************************************
+
+//#define LOG_GENERAL (1U <<  0)
```

Comment:
You still need to define `LOG_GENERAL` here or you can't enable it because you're supposed to define `VERBOSE` before including `logmacro.h`.  If you define `LOG_GENERAL` here, `logmacro.h` won't duplicate it.

Reply:
ok, I'll fix